pkgs/ffigen/test/native_objc_test/automated_ref_count_test.dart

```diff
@@ -328,9 +477,10 @@ void main() {
       // is working correctly by holding a reference to an object returned by a
       // method, after the original wrapper object is gone.
       final counter = calloc<Int32>();
-      unownedReferenceInner(counter);
+      final obj1bRaw = unownedReferenceInner(counter);
       doGC();
       expect(counter.value, 0);
+      expect(objectRetainCount(obj1bRaw), 0);
       calloc.free(counter);
     });
   });
```

Comment:
Please add a test that exercises the behavior above 255. (And define and document the behavior, also see the other comment.)

Reply:
Done